src/app/books/books.service.ts

```diff
@@ -0,0 +1,28 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { map, delay } from 'rxjs/operators';
+import { Book } from './book/book.model';
+
+const API_URL =
```

Comment:
type this const (also tip: you can create separate config file where you will declare all constants you will use in you project)

Reply:
I thought because of Type Inference I don't need to write it there. I will amend it.